File: Manual 06/Question04.cpp
#include <iostream>
using namespace std;

// Function to print all prime numbers between two boundaries
void Prime(int x, int y)
{
    for (int i = x; i <= y; i++)
    {
        int count = 0;
        for (int j = 1; j <= i; j++)
        {
            if (i % j == 0)
            {
                count++;
            }
        }
        if (count == 2)
        {
            cout << i << "  ";
        }
    }
    cout << endl;
}

int main()
{
    int lb, hb;
    cout << "Enter Lower Boundary: ";
    cin >> lb;
    cout << "Enter Upper Boundary: ";
    cin >> hb;
    Prime(lb, hb);
    return 0;
}

File: Python/Manual 06/Question04.py
num1 = int(input("Enter Lower Limit: "))
num2 = int(input("Enter Upper Limit: "))

count = 0
for i in range(num1, num2 + 1):
    count = 0
    for j in range(2, i):
        if not (i % j):
            count += 1
    if not count:
        print(str(i) + " is a Prime Number")
